DataFormats/GEMDigi/src/classes_def.xml

```diff
@@ -9,29 +9,37 @@
 <class name="MuonDigiCollection<GEMDetId,GEMDigi>"/> 
 <class name="edm::Wrapper<MuonDigiCollection<GEMDetId,GEMDigi> >" splitLevel="0"/> 
 
-<class name="GEMVfatStatusDigi" ClassVersion="3">
- <version ClassVersion="3" checksum="569767420"/>
+<class name="GEMVfatStatusDigi" ClassVersion="0">
+ <version ClassVersion="0" checksum="1981269550"/>
 </class>
 <class name="std::vector<GEMVfatStatusDigi>"/>
-<class name="std::map<uint16_t,std::vector<GEMVfatStatusDigi> >"/>
-<class name="MuonDigiCollection<uint16_t,GEMVfatStatusDigi>"/> 
-<class name="edm::Wrapper<MuonDigiCollection<uint16_t,GEMVfatStatusDigi> >" splitLevel="0"/> 
+<class name="std::map<GEMDetId,std::vector<GEMVfatStatusDigi> >"/>
+<class name="MuonDigiCollection<GEMDetId,GEMVfatStatusDigi>"/> 
+<class name="edm::Wrapper<MuonDigiCollection<GEMDetId,GEMVfatStatusDigi> >" splitLevel="0"/> 
 
-<class name="GEMGEBStatusDigi" ClassVersion="3">
```

Comment:
were any of these removed classes ever actually used in production?

Reply:
no.
they are only used in DQM when we need a lot more details